check_terms/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "check_terms"
```

Comment:
I see we created a new crate, "check_terms" here. 
We already have a root crate "terms". Instead of creating a new "check_terms" crate, can we implement the new features in the root crate "terms"?

The root module is defined here.
https://github.com/EtherTW/Terms/blob/4baa03f12f6dbd17fde54db7b82703530be57dc4/Cargo.toml#L1-L10